app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -403,6 +410,25 @@ public void onClick(View v) {
 				//Log.e("Flash",preview.camera.getParameters().getSupportedFlashModes().get(0));
 			}
 		});
+
+		grid_overlay_button = (ImageButton)view.findViewById(R.id.grid_overlay);
+		grid_overlay_button.setLayoutParams(parmFlash);
+		grid_overlay_button.setImageResource(R.drawable.grid_button);
```

Comment:
right click on `res` --> New --> Vector Asset --> Click on Android icon and search Grid